README.md

```diff
@@ -1,22 +1,21 @@
-# Language Model Evaluation Harness
+# Promptsource X Language Model Evaluation Harness
 
 ![](https://github.com/EleutherAI/lm-evaluation-harness/workflows/Build/badge.svg)
 [![codecov](https://codecov.io/gh/EleutherAI/lm-evaluation-harness/branch/master/graph/badge.svg?token=JSG3O2427J)](https://codecov.io/gh/EleutherAI/lm-evaluation-harness)
 
 ## Overview 
 
-This project provides a unified framework to test autoregressive language models (GPT-2, GPT-3, GPTNeo, etc) on a large number of different evaluation tasks.
+This project provides a unified framework to test language models (GPT-2, GPT-3, GPTNeo, etc) and seq2seq (T5, T0) models via prompt evaluation. The prompts are provided via `promptsource`; all datasets are in huggingface `datasets`.
 
-Features:
+This fork is not backwards compatible with the original evaluation harness.
 
-- 200+ tasks implemented
```

Comment:
I don't see a reason to remove this

Reply:
The original set of tasks aren't set up to work currently. We kept them in a deprecated file for reference. I added a note on about how many (higher-level, glue = 1) tasks are currently integrated.